### DISCUSSION TITLE
download stops at near end and ask for password and username -- screenshot inside

### DISCUSSION THREAD

**Discussion by hemazone** [comfyanonymous/ComfyUI#4594] [Q&A]
![Untitl1ed](https://github.com/user-attachments/assets/4c333a1e-3de5-49bc-85e5-3fb5efe3195c)
hello ,

well i tried to download ComfyUI_windows_portable_nvidia.7z ( 1.47 GB )
and when it reached to the end of downloading it stop and asked me for username and password
and no mutter i tried to give it it won't start and pop up again asking username and password
i tried to download again and its same results

**Comment by ltdrdata** [comment]
Download via browser instead of 3rd party downloader.
And this is a github issue. It's not related to ComfyUI.
